### PR TITLE
Update Companies with Remote DNA section

### PR DESCRIPTION
**Pull request by miketwo**
- Added descriptions for the first 20ish companies
- Removed Everlane (no mention of remote work on their site or on glassdoor, and this: https://twitter.com/everlane/status/527975807126216704)

### CODE REVIEW

**Comment by waldyrious** [comment]
Nice work! :+1:

File: README.md [hunk 1]
```diff
@@ -123,29 +123,28 @@ A curated list of awesome [remote working](http://en.wikipedia.org/wiki/Telecomm
   2. [Wide Teams](http://www.wideteams.com) - Each episode a one-on-one interview with a remote worker taking about workflow and practices
 
 ## Companies with "remote DNA"
-  1. [Articulate](https://www.articulate.com/company/careers.php)
-  1. [Automattic](http://automattic.com/work-with-us/)
-  1. [BaseCamp](https://basecamp.com/team)
+  1. [Articulate](https://www.articulate.com/company/careers.php) - EdTech. Makes software that helps teachers make e-learning courses. Ruby. Node. C# and .NET for Windows app.
+  1. [Automattic](http://automattic.com/work-with-us/) - Makers of WordPress.com and  Gravatar.
```
Comment (lukasz-madon) [inline README.md]: extra space

File: README.md [hunk 2]
```diff
@@ -123,29 +123,28 @@ A curated list of awesome [remote working](http://en.wikipedia.org/wiki/Telecomm
...
-  1. [Compose](https://compose.io)
-  1. [DataStax](http://www.datastax.com/company/careers)
-  1. [DuckDuckGo](https://duck.co/help/company/hiring)
-  1. [Etsy](https://www.etsy.com/careers)
-  1. [Everlane](https://www.everlane.com/jobs)
-  1. [Eyeo](https://eyeo.com/jobs)
-  1. [Fog Creek](http://www.fogcreek.com/careers/)
-  1. [Ghost](https://ghost.org/careers/)
-  1. [Gitbook] (http://jobs.gitbook.com/)
+  1. [CircleCI](https://circleci.com/jobs#engineer) - Provides continouos integration tools and services.
```
Comment (lukasz-madon) [inline README.md]: continuous

**Comment by lukasz-madon** [comment]
:+1:

**Comment by lukasz-madon** [comment]
@miketwo I'd be great if you could update it, then I'll merge it

**Comment by miketwo** [comment]
@lukasz-madon I made the fixes you specified. Also added a few additional descriptions. lmk if you'd like anything else done to this branch.

**Pull request merged**